### PR TITLE
Move talk-creation to TalkController

### PR DESCRIPTION
**Pull request by heiglandreas**
This PR moves talk creation from the Events-Controller to the Talks
controller and adapts the routing appropriately.

Additionally tests for minimum-talk creation have been added to make
sure talk-creation works as expected.

This is the first of a series of PRs that will follow to enable talk-editing via API and addresses issue JOINDIN-525

### CODE REVIEW

File: src/controllers/TalksController.php [hunk 1]
```diff
@@ -189,4 +189,95 @@ protected function getTalkById($db, $request, $talk_id, $verbose = false)
...
+     * @throws Exception
+     * @return array|bool
+     */
+    public function createTalkAction(Request $request, PDO $db)
+    {
+        if(!isset($request->user_id)) {
+            throw new Exception("You must be logged in to create data", 400);
+        }
+
+            $talk['event_id'] = $this->getItemId($request);
```
Comment (chrishoult) [inline src/controllers/TalksController.php]: Funny tabbing here - I'm pretty sure we've standardized on 4 spaces

Comment (chrishoult) [inline src/controllers/TalksController.php]: Ah, C&P from the other controller, where it was in a `switch` :)

**Comment by chrishoult** [comment]
Other than the style comment above, looks like it's good; I've not tested yet.

**Comment by akrabat** [comment]
Merged: e5f2d8b0e410c11673957aa2de23722351d4fa37

**Comment by akrabat** [comment]
@chrishoult: I fixed the code style issues on merge.

**Pull request closed without merging**